### PR TITLE
feat(it): test validating a request is sent on page load

### PR DESCRIPTION
**Pull request by francoislg**
[COM-800]

This here adds a simple HTML page served by the lightweight node package `http-server`, and a Cypress test that validates that the outgoing request and response are correct.

I think we can do more than that, but I felt this was a pretty good start :smile: 

We already have a Jenkins job running that should notify us if the thing is failing.

I tried to design the thing to allow logging to any environment, and most importantly, try any version. So in the long run, we could use this as a certifier in the deployment pipeline with a deployed `MAJOR_MINOR_PATCH` version before deploying to the `MAJOR`!

[COM-800]: https://coveord.atlassian.net/browse/COM-800

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```
Comment (Spuffynism) [inline cypress/plugins/index.js:1]: Do we need this file?

**Review by Spuffynism — APPROVED** [review]
I'm **so** glad we have this. Nice work! :rocket:

File: cypress/plugins/index.js (hunk 1 above)
Comment (francoislg) [reply cypress/plugins/index.js:1]: The files are needed for Cypress, even if empty, yeah

Comment (Spuffynism) [reply cypress/plugins/index.js:1]: Huh, alright then! something something green boat

**Review by Spuffynism — APPROVED** [review]

**Review by louis-bompart — APPROVED** [review]
Looks good :D

**Review by toussaints — APPROVED** [review]

**Review by cnguyen-coveo — APPROVED** [review]

**Pull request merged**